### PR TITLE
src/components/AppBar/index.jsx added login button

### PR DESCRIPTION
**Pull request by Med16-11**
- login button is now available for users that will redirect them to authorize using github auth.
image

### CODE REVIEW

File: src/components/AppBar/index.jsx [hunk 1]
```diff
@@ -78,10 +81,22 @@ export default function AppBar(props) {
             Pulpito
           </RouterLink>
         </Typography>
+        <Button
+          component={RouterLink}
+          to="http://localhost:8082/login" 
+          color="inherit"
+          sx={{
+            bgcolor: "#b81f14",
```
Comment (kamoltat) [inline src/components/AppBar/index.jsx:89]:
```suggestion

```
Think color Red doesn't really fit well with our theme, we can remove it.

**Review by kamoltat — CHANGES_REQUESTED** [review]
Just minor change

File: src/components/AppBar/index.jsx (hunk 1 above)
Comment (Med16-11) [reply src/components/AppBar/index.jsx:89]: Sure, any colour suggestions that we can use here?

Comment (kamoltat) [reply src/components/AppBar/index.jsx:89]: I think just deleting the line I suggested will do the trick :)

**Comment by kamoltat** [comment]
Closing this pull request, since we already satisfied this feature request.

**Pull request closed without merging**